### PR TITLE
th-55: Add clsx package for applying multiple classes

### PR DESCRIPTION
**Pull request by denred**
- Add helper to apply multiple classes using the clsx library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
   "simple-git-hooks": {
     "pre-commit": "npx lint-staged",
     "commit-msg": "npx commitlint --edit $1"
+  },
+  "dependencies": {
+    "clsx": "2.0.0"
```
Comment (kalinkaaaa14) [inline package.json]: should be on fe package.json

File: frontend/src/libs/helpers/classname-helper.ts
Comment (kalinkaaaa14) [inline frontend/src/libs/helpers/classname-helper.ts:1]: should be separate folder get-valid-classnames/get-valid-classnames.helper -- view example of the structure in shared

File: frontend/src/libs/helpers/classname-helper.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import clsx from 'clsx';
+
+const getClassNames = (...classNames: string[]): string => {
```
Comment (kalinkaaaa14) [inline frontend/src/libs/helpers/classname-helper.ts]: add 
```suggestion
const getValidClassNames = (...classNames: ClassValue[]): string => {
```

**Review by kalinkaaaa14 — CHANGES_REQUESTED** [review]

**Review by PavloSukhinin — APPROVED** [review]

**Review by Druxight — APPROVED** [review]

**Review by kalinkaaaa14 — CHANGES_REQUESTED** [review]
package.json packages should not be changed manually, use npm

**Pull request merged**